src/lrs_annotation/scripts/svs/sv_vcf_modifier.py

```diff
@@ -128,22 +141,17 @@ def modify_sniffles_vcf(file_in: str, file_out: str, fa: str, sex_mapping_file:
                 print(f'Lines processed: {index}')
 
             # alter the sample line in the header
-            if line.startswith('#'):
-                if 'FORMAT=<ID=GT' in line:
-                    f_out.write('##FORMAT=<ID=RD_CN,Number=1,Type=Integer,Description="Copy number of this variant">\n')
-                # Fix the GQ type, as Jasmine merged SV VCFs have GQ as a string when it should be an integer
-                if '##FORMAT=<ID=GQ,Number=1,Type=String' in line:
-                    f_out.write('##FORMAT=<ID=GQ,Number=1,Type=Integer,Description="The genotype quality">\n')
-                    continue
-                if 'CHROM' in line:
-                    # Find the sample IDs in the header line to match to the input sex values
-                    l_split = line.rstrip().split('\t')
-                    sample_ids = l_split[9:]
-
+            if line.startswith('#') and 'CHROM' not in line:
+                f_out.write(process_header_line(line))
+                continue
```

Comment:
The previous code would add a format header for `ID=RD_CN` alongside the `ID=GT` format header, but this drops the existing `ID=GT` line.